src/app/gui/glLUTwidget.cpp

```diff
@@ -781,7 +783,8 @@ void GLLUTWidget::glDrawSlice(Slice * s) {
     }
     glEnable(GL_COLOR_LOGIC_OP);
     //glLogicOp(GL_COPY_INVERTED);
-    glLogicOp(GL_XOR);
+    //glLogicOp(GL_XOR);
+    glLogicOp(GL_EQUIV);                //fix drawing inverted color (2/2/16)
```

Comment:
not sure why the original inversion got commented out, but this line restores the ability to see the LUT marked up with a sample drawn from the active image (now they are visualized with small white 'x's as I think was originally intended)